### DISCUSSION TITLE
PostgreSQL activity it spins up externally mounted disks every hour

### DISCUSSION THREAD

**Discussion by TommasoCostanzo** [nextcloud/all-in-one#4218] [Questions] [answered]
It is almost a year that I am very happily using Nextcloud AIO, and I want to thank you the entire team for the great work. 

Recently I activated the activity apps and my external mounted local disks where spinning up every 5 minutes. However, that make sense if one want to keep track of what happens. Anyway, now that I have deactivated the app everything seems like before. In any case there is still some Postgres activity that start roughly every hour and it also spins up the disks. I have tried to check if there is a parameter in config.php or anything like that, but I did not find any obvious parameter controlling the database synchronization. The only parameter that seems related is "files_no_background_scan", but by default should be every 10 minutes that does not match with the ~1 hour gap I see. Also the regular cron job that run every 5 minutes does not wake up my disks. 

Since PostgreSQL is not the default of NC but only of AIO, I thought the question should be addressed here, and I  would like to ask if there is any setting where I can change the PostgreSQL scan/synchronization from 1 hour to maybe 6 hours (since I do not have many file changing per hour ~10 additional photo per day and all uploaded at the same time). Is that possible? Any suggestion is very welcome.

Thank you in advance

**Comment by szaimen** [comment, marked as answer]
Hi, I am pretty sure that this is caused by Nextcloud itself and not by AIO's implementation.

So feel free to create a new thread at https://help.nextcloud.com in order to get community help on the topic.

**Reply by TommasoCostanzo** [reply]
Thank you @szaimen I have been investigating the logs bit more as well as to run cron.php manually. It looks like the cron.php is not always spinning up the disk, and I guess it depend from some additional condition like Redis cache or similar because from the logs I see irregular timings (some due to the users, but others I think are just for the background housekeeping task of the trash bin, cache etc.). I found also this discussion in NC community [https://help.nextcloud.com/t/random-hdd-spin-up-after-being-idle/58348](url). However, I am not sure how the cronjob is run in the docker container, especially if I would like to implement something similar to this approach [https://forum.turris.cz/t/nextcloud-spun-down-drives/15784?u=th1j5](url). Is there any way to remove the cronjob in the docker and setting a cronjob on the base os that run the command "php /var/www/html/cron.php" inside the container. Because if I can trigger it from my OS I might also think to use a python script or similar, that will give me more control on when to run the cron.php based on multiple conditions like if the disk is active or idle.
